### ISSUE TITLE
The table and heatmap display of neighbors should include the corr. coeff. value

### ISSUE THREAD

**Issue by sdash-github** [legumeinfo/lis_expression#17]
This was the feature in the old neighbors display. Lost this feature while coding to handle the new ajax. Should bring it back.  (May involve substantial code change by modifying draw functions and the y-axis array for heatmap).

**Comment by sdash-github** [comment]
The neighbors table display now shows correlation and the neighbor uniquename as a link.
But it is buggy.
Next step: Attempt in heat map and then clean bugs.

**Comment by sdash-github** [comment]
1.  Now shows up in heatmap but ... ... ...
Bug:
After correlation and anchor-text is used for the y-axis; it doesn't pick up the top neighbors in the text-box(input box)

**Comment by sdash-github** [comment]
The above was in top-neighbors.  
Found out:  to display anchor texts in selected correlation list, needs more code change; haven't yet figured out code flow and where to change. (Next TO DO)

**Notes for picking up tomorrow:**
lis_expr_jsFunctions.js
line 928: how do I get newNeighbors # can't find
lines 940-942 has neighborUniquenamesList and newCorrelationsList
(this is where to generate the anchored list)

**Comment by sdash-github** [comment]
Now anchored text with corr is displayed okay for selected-corr-gene-list.
BUG:
Once selected-corr-gene-list is selected, the top-neighbors don't respond.

**Comment by sdash-github** [comment]
Created a separate set of variable names for neighbor-uniquenames, corr, exprsn-values for the selected-corr list and then it worked.
Worked, tested for 3 spp.
